frontend/fingertips-frontend/README.md

```diff
@@ -119,3 +127,35 @@ These server pages are responsible for making any calls to fetch any data. Then
 The page components are pure react components and must have the `use client` directive at the top. This is needed for the purpose of using the `govuk-react` component library. This library uses `styled-components` and react hooks and therefore need to be client components.
 
 However, Next.js will still use these component to render the page server-side.
+
+## Auto generate scripts
+
+CAUTION: Running these scripts will overwrite previously generated code.
```

Comment:
I think it's perhaps worth adding another sentence here to say that you would only need to do it when you've updated the API contract and want to re-generate the frontend's API clients and mocks.